pkgs/development/tools/makerpm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, zlib, libarchive, openssl }: 
+
+stdenv.mkDerivation { 
+  name = "makerpm-1.0";
+  version = "1.0";
+
+  builder = ./builder.sh;
+
+  buildInputs = [ zlib libarchive openssl ];
+
+  src = fetchFromGitHub {
+    owner = "ivan-tkatchev";
+    repo = "makerpm";
+    rev = "141a4444b9bcb1df27ee33c4e0855ab58128017e";
+    sha256 = "1q76zv0qs5gx9w0qfd64kldw9gngafr16izh0prvw4cd8kn4cqrk";
+  };
+
+  meta = {
```

Comment:
You can use `meta = with stdenv.lib; {` here, so you don't have to prefix the attributes like licenses and platforms.